packages/datadog-instrumentations/src/child_process.js

```diff
@@ -0,0 +1,176 @@
+'use strict'
+
+const util = require('util')
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const childProcessChannelStart = channel('datadog:child_process:execution:start')
+const childProcessChannelFinish = channel('datadog:child_process:execution:finish')
+const childProcessChannelError = channel('datadog:child_process:execution:error')
+
+// ignored exec method because it calls to execFile directly
+const execAsyncMethods = ['execFile', 'spawn']
```

Comment:
no fork ?

Reply:
Fork is not a command, creates a new thread starting a new module, but it is not executing a command, which is what we want to trace.

Reply:
I just checked and it uses spawn() internally anyway